app/models/main.py

```diff
@@ -1687,6 +1714,16 @@ def serialize(self):
 
         return purge_nulls_from_data(data)
 
+    def _parent_brief_fields(self, parent_brief):
```

Comment:
It might be nicer to name this in a way that makes it clear it's related to serialization. And also it's often nice to name "getter" methods with "get", so maybe prefixed with `_get_...`

Reply:
Yep, maybe `_get_serialized_parent_brief_fields` is better in this case.